src/content/packages-and-plugins/swift-package-manager/for-app-developers.md

```diff
@@ -0,0 +1,144 @@
+---
+title: Swift Package Manager for app developers
+description: How to use Swift Package Manager for native iOS or macOS dependencies
+---
+
+:::warning
+Flutter is migrating to [Swift Package Manager][] to manage iOS and macOS native
+dependencies.
+This is an experimental feature that might change in the future.
+It is currently only available on the [`main` channel][].
+Flutter will continue to support CocoaPods until further notice.
```

Comment:
**issue:** Condition, then instruction per [GSG](https://developers.google.com/style/sentence-structure).
Keep the documentation timeless per [GSG](https://developers.google.com/style/timeless-documentation).

```suggestion
To manage iOS and macOS native dependencies,
Flutter is migrating to [Swift Package Manager][] .
Flutter's support of SPM as described in this guide is under development. 
The implementation might change in the future.
SPM support can be found on the [`main` channel][] only.
Flutter continues to support CocoaPods.
```

Reply:
Normally I agree that "condition, then instruction" is better, but I think it makes the first sentence choppy. I personally prefer the original first sentence. How strongly do you feel about this?

I updated the remaining sentences!